bun-tests/fake-snippets-api/fixtures/get-test-server.ts

```diff
@@ -76,29 +82,54 @@ const seedDatabase = (db: DbClient) => {
       phone: "555-123-4567",
     },
   })
+
   const account2 = db.addAccount({
     github_username: "jane",
   })
-  const order = db.addOrder({
-    account_id: account.account_id,
-    is_running: false,
-    is_started: false,
-    is_finished: false,
-    error: null,
-    has_error: false,
-    circuit_json: [
+
+  let circuit_json = undefined
+  if (options.withCircuitJson) {
+    circuit_json = [
       {
         type: "source_component",
         ftype: "simple_resistor",
         source_component_id: "source_component_1",
         name: "R1",
```

Comment:
We can actually build the circuit json on the fly using eval

Reply:
Thrn we dont need the flag. If it wrrors while bug iglding, thats fine! We have errors for that on package releases that would be good to test!